src/test/java/util/xcmailr/util/SendRequest.java

```diff
@@ -10,19 +10,18 @@
 import org.apache.http.client.entity.UrlEncodedFormEntity;
 import org.apache.http.client.methods.HttpGet;
 import org.apache.http.client.methods.HttpPost;
-import org.apache.http.impl.client.DefaultHttpClient;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
 import org.apache.http.message.BasicNameValuePair;
 
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
 import com.jayway.jsonpath.JsonPath;
 
-import net.minidev.json.JSONArray;
-import net.minidev.json.JSONObject;
-import net.minidev.json.parser.JSONParser;
-import net.minidev.json.parser.ParseException;
-
 public class SendRequest
 {
-    private static DefaultHttpClient httpClient = new DefaultHttpClient();
+    private static CloseableHttpClient httpClient = HttpClientBuilder.create().build();
```

Comment:
This may create a `peer not authorized` exception. Please look into why this is thrown but not with the `DefaultHttpClient`.